modules/va_facilities/spec/requests/nearby_request_spec.rb

```diff
@@ -31,75 +34,86 @@ def query_params(url)
 
   context 'when requesting JSON API format' do
     it 'responds to GET #index' do
-      get base_query_path + address_params, params: nil, headers: accept_json
-      expect(response).to be_success
-      expect(response.body).to be_a(String)
-      json = JSON.parse(response.body)
-      expect(json['data'].length).to eq(21)
-      # expect(json['meta']['distances']).to eq([])
-    end
-
-    xit 'responds with pagination links' do
-      # TODO: links object not currently being returned
-      get base_query_path + address_params, params: nil, headers: accept_json
-      expect(response).to be_success
-      json = JSON.parse(response.body)
-      expect(json).to have_key('links')
-      links = json['links']
-      expect(links).to have_key('self')
-      expect(links).to have_key('first')
-      expect(links).to have_key('last')
-      expect(links).to have_key('prev')
-      expect(links).to have_key('next')
+      setup_pdx
+      VCR.use_cassette('bing/isochrone/pdx_drive_time_60') do
+        get base_query_path + address_params, params: nil, headers: accept_json
+        expect(response).to be_success
+        expect(response.body).to be_a(String)
+        json = JSON.parse(response.body)
+        expect(json['data'].length).to eq(10)
+        # expect(json['meta']['distances']).to eq([])
```

Comment:
Is this supposed to be commented out?

Reply:
I've actually been going waffling on whether this response needs to include the `distances` array, so I commented it out (because otherwise the test fails), but didn't want to remove it completely and forget about it.

My current thought is return the `distances` array (it will always be empty for this endpoint) in the name of consistency across facilities endpoints.

Reply:
Now returning an empty `distances` array a part of `metadata`.